lib/trailblazer/operation/controller/active_record.rb

```diff
@@ -7,6 +7,10 @@ def setup_operation_instance_variables!
   end
 
   def operation_model_name
+    # set the right variable name if collection
+    if @model.class.to_s.match(/ActiveRecord_Relation\z/)
```

Comment:
A suitable compromise would be `@operation.is_a?(Trailblazer::Operation::Collection)`.